homeassistant/components/xiaomi_miio/button.py

```diff
@@ -0,0 +1,149 @@
+"""Support for Xiaomi buttons."""
```

Comment:
Please split off the different platforms to separate PRs. We don't need to add all platform support in the same PR for these devices. This PR is way too big.

Start with a platform that is already existing and add support to that platform for the new device. When that PR is merged, create the next PR etc. We need to do this in small steps. Thanks!